mapserver/django/django_project/fba/management/commands/ingest_hazard_data.py

```diff
@@ -99,11 +98,11 @@ def request_data(self):
 
                 if not hazard_classes.exists():
                     all_hazard_class = HazardClass.objects.all()
-                    hazard_class_last_id = all_hazard_class[all_hazard_class.count() - 1].id + 1
+                    hazard_class_last_id = all_hazard_class[-1].id + 1
```

Comment:
Fix the syntax. But we should define the classes first and put it in the table as fixtures or so. Not consume it in runtime.